### PR TITLE
Ignition to Gazebo migration

### PR DESCRIPTION
**Pull request by marcoag**
Work towards https://github.com/osrf/lrauv/issues/234.

Modifications needed for the Ignition to Gazebo name changing migration.

Signed-off-by: Marco A. Gutierrez <marco@openrobotics.org>

### CODE REVIEW

**Comment by marcoag** [comment]
Thanks for the check!

`./README.md`

This one is left for the last update as shown in https://github.com/osrf/lrauv/issues/234.

```
./docker/empty_world/Dockerfile
./docker/tests/Dockerfile
./docker/base/Dockerfile
./docker/debug_integration/Dockerfile
```

These will also come in later update as part of the mbari side update, see https://github.com/osrf/lrauv/issues/234.

`./lrauv_system_tests/src/LRAUVController.cc` 

This one has [a line here](https://github.com/osrf/lrauv/blob/7b433fb137d5f2d3543e7655e6bb4560836a914c/lrauv_system_tests/src/LRAUVController.cc#L103) that might depend on docker changes, so I left it until docker changes are done.

```
./lrauv_system_tests/worlds/buoyant_tethys_at_depth.sdf
./lrauv_system_tests/worlds/buoyant_tethys.sdf
./lrauv_description/models/tethys_equipped/model.sdf
```

Updated these and some extra ones I found on the way.

**Review by hidmic — COMMENTED** [review]
LGTM. We need to update the `lrauv-application` side and the wiki now.

File: lrauv_gazebo_plugins/models/turbidity_generator/model.sdf [hunk 1]
```diff
@@ -5,9 +5,9 @@
 -->
 <sdf version="1.6">
 
-  <!-- Requires ParticleEmitter2 in ign-gazebo 4.8.0, which will be copied
-    to ParticleEmitter in Ignition G.
-    See https://github.com/ignitionrobotics/ign-gazebo/pull/730 -->
+  <!-- Requires ParticleEmitter2 in gz-sim 4.8.0, which will be copied
+    to ParticleEmitter in Gazebo Ggarden.
```
Comment (hidmic) [inline lrauv_gazebo_plugins/models/turbidity_generator/model.sdf:9]: @marcoag nit: 
```suggestion
    to ParticleEmitter in Gazebo Garden.
```

Or back to `Gazebo G.`. I see both in other SDF files below.

**Review by hidmic — APPROVED** [review]
CI is happy so I'm happy.

**Pull request merged**